server/src/main/java/com/cloud/api/query/dao/HostJoinDaoImpl.java

```diff
@@ -321,7 +321,7 @@ public HostForMigrationResponse newHostForMigrationResponse(HostJoinVO host, Enu
                 hostResponse.setMemoryTotal(host.getTotalMemory());
                 Float memWithOverprovisioning = host.getTotalMemory() * ApiDBUtils.getMemOverprovisioningFactor(host.getClusterId());
                 hostResponse.setMemWithOverprovisioning(decimalFormat.format(memWithOverprovisioning));
-                hostResponse.setMemoryAllocated(decimalFormat.format((float) mem / memWithOverprovisioning * 100.0f) +"%");
+                hostResponse.setMemoryAllocated(mem);
 
```

Comment:
Will this cause regression?

Reply:
It will but we can introduce two new fields `memoryallocaedpercentage` & `cpuallocaedpercentage` to compensate.
This will ensure that the response for both HostResponse and HostForMigrationResponse will be the same. Sounds good @rhtyd ?